facebook/OpenBIC
# Artemis: cb: Support switch error event

# Description
- Support switch CCR cyctem error event.
- Support ACCL card power good fail event.

# Motivation
- Support switch CCR cyctem error event.
- Support ACCL card power good fail event.

# Test Plan
- Build code: Pass
- Check ACCL card power good fail event: Pass
- Check switch CCR cyctem error event: Pass

# Log
- Check ACCL card power good fail event ......
0    all      2018-03-09 06:31:12    pldmd            State Sensor: CB_SENSOR_ACCL_1, power good fail
0    all      2018-03-09 06:31:13    pldmd            State Sensor: CB_SENSOR_ACCL_2, power good fail
0    all      2018-03-09 06:31:14    pldmd            State Sensor: CB_SENSOR_ACCL_3, power good fail
0    all      2018-03-09 06:31:15    pldmd            State Sensor: CB_SENSOR_ACCL_4, power good fail
0    all      2018-03-09 06:31:16    pldmd            State Sensor: CB_SENSOR_ACCL_5, power good fail
0    all      2018-03-09 06:31:17    pldmd            State Sensor: CB_SENSOR_ACCL_6, power good fail
0    all      2018-03-09 06:31:18    pldmd            State Sensor: CB_SENSOR_ACCL_7, power good fail
0    all      2018-03-09 06:31:19    pldmd            State Sensor: CB_SENSOR_ACCL_8, power good fail
0    all      2018-03-09 06:31:20    pldmd            State Sensor: CB_SENSOR_ACCL_9, power good fail
0    all      2018-03-09 06:31:21    pldmd            State Sensor: CB_SENSOR_ACCL_10, power good fail
0    all      2018-03-09 06:31:22    pldmd            State Sensor: CB_SENSOR_ACCL_11, power good fail
0    all      2018-03-09 06:31:23    pldmd            State Sensor: CB_SENSOR_ACCL_12, power good fail
......

- Check switch CCR cyctem error event ......
0    all      2018-03-09 06:27:45    pldmd            Device type: CB_PESW_0 (0x11), board info: MAIN_SOURCE (0x0), event type: SYSTEM_ERROR (0x6) Assert event
0    all      2018-03-09 06:27:45    pldmd            Device type: CB_PESW_0 (0x11), board info: MAIN_SOURCE (0x0), event type: PEX_FATAL_ERROR (0x7) Assert event
0    all      2018-03-09 06:27:45    pldmd            Device type: CB_PESW_1 (0x12), board info: MAIN_SOURCE (0x0), event type: POR_BISR_TIMEOUT (0x8) Assert event
0    all      2018-03-09 06:27:46    pldmd            Device type: CB_PESW_1 (0x12), board info: MAIN_SOURCE (0x0), event type: FLASH_SIGNATURE_FAIL (0x9) Assert event
......
